test/regression/expected/type_support.out

```diff
@@ -308,6 +308,27 @@ SELECT * FROM json_tbl;
  {}
 (4 rows)
 
+-- BLOB
+CREATE TABLE blob_tbl(a bytea);
+INSERT INTO blob_tbl SELECT CAST(a as bytea) FROM (VALUES
+    ('\x'),
+    ('\x110102030405060708090a0b0c0d0e0f'),
+    (''),
+    ('\x00'),
+    ('\x07'),
+    (NULL)
+) t(a);
+SELECT * from blob_tbl;
+                 a                  
+------------------------------------
+ \x
+ \x110102030405060708090a0b0c0d0e0f
+ \x
+ \x00
+ \x07
+ 
+(6 rows)
+
```

Comment:
Can you add a comparison test too? Something like:

SELECT * FROM blob_tbl WHERE a = '\x00';